test-series/1/test-1.0.0.ipynb

```diff
@@ -0,0 +1,1449 @@
+{
```

Comment:
<p>A different choice of window function might provide a broader range that's EoR-dominated (for instance, the Dolph-Chebyshev window).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera-validation/pull/51/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>That's a good point, and we should use that in future tests (or bh7), but it doesn't affect the qualitative results here. This test isn't really about "can we detect the EoR", rather whether where we do it's consistent with the input.</p>